Source/DafnyDriver/Commands/GenerateTestsCommand.cs

```diff
@@ -117,6 +117,7 @@ private enum Mode {
     dafnyOptions.ForbidNondeterminism = true;
     dafnyOptions.DefiniteAssignmentLevel = 2;
     dafnyOptions.UseBaseNameForFileName = false;
+    dafnyOptions.EmitUncompilableCode = false;
```

Comment:
This should be the default. I think you don't need to do this if you read the value straight from the `Option` instead.

Reply:
Even if this is the default, setting it here enables to remember that it's there. The same can be said for `UseBaseNameForFileName` so I keep it for consistency.